appassembler-maven-plugin/src/it/booterLicenseHeaderTest/verify.groovy

```diff
@@ -58,7 +58,7 @@ t.checkExistenceAndContentOfAFile(new File( fileBinFolder, "booter-windows/bin/b
     '@REM ******************************************',
     '@REM --- This is my own license header file ---',
     '@REM ******************************************',
-    '%JAVACMD% %JAVA_OPTS%  -classpath %CLASSPATH% -Dapp.name="booterLicenseHeaderTest" -Dapp.repo="%REPO%" -Dapp.home="%BASEDIR%" -Dbasedir="%BASEDIR%" org.codehaus.mojo.appassembler.booter.AppassemblerBooter %CMD_LINE_ARGS%',
+    '%JAVACMD%  %JAVA_OPTS% -classpath %CLASSPATH% -Dapp.name="booterLicenseHeaderTest" -Dapp.repo="%REPO%" -Dapp.home="%BASEDIR%" -Dbasedir="%BASEDIR%" org.codehaus.mojo.appassembler.booter.AppassemblerBooter %CMD_LINE_ARGS%',
```

Comment:
Special reason why this has been changed? also Line 39 ?

Reply:
i must've been confused - removed those changes.

Reply:
oops - de ja vu :) https://github.com/mojohaus/appassembler/pull/58#issuecomment-290191048 - some of these `mvn --show-version --errors --batch-mode -Prun-its clean verify` tests failed

 ```
[INFO] Building: booterLicenseHeaderTest/pom.xml
[INFO] run script verify.groovy
[INFO] ..FAILED (2.3 s)
[INFO]   The post-build script did not succeed. The expected content 'exec "$JAVACMD" $JAVA_OPTS  \' in the file '/Users/nezda/code/appassembler.git/appassembler-maven-plugin/target/it/booterLicenseHeaderTest/target/generated-resources/appassembler/booter-unix/bin/booterLicenseHeaderTest' couldn't be found.
```

```
[INFO] Building: programLicenseHeaderTest/pom.xml
[INFO] run script verify.groovy
[INFO] ..FAILED (2.2 s)
[INFO]   The post-build script did not succeed. The expected content 'exec "$JAVACMD" $JAVA_OPTS  \' in the file '/Users/nezda/code/appassembler.git/appassembler-maven-plugin/target/it/programLicenseHeaderTest/target/appassembler/bin/programLicenseHeader-test' couldn't be found.
```

reverting that last change

Reply:
The travis test failure appears to be because they no longer support Java 7

https://travis-ci.org/github/mojohaus/appassembler/jobs/679529347

![image](https://user-images.githubusercontent.com/179091/80309229-4ecc9d80-8799-11ea-9ca1-f17b4f259497.png)

Reply:
What can I do to get this merged?